x-pack/test/api_integration/apis/endpoint/alerts/index.ts

```diff
@@ -260,134 +212,6 @@ export default function ({ getService }: FtrProviderContext) {
         expect(emptyBody.alerts.length).to.eql(0);
       });
 
-      it('alerts api should return data using `before` by custom sort parameter, descending', async () => {
-        const { body } = await supertest
-          .get(
-            `/api/endpoint/alerts?${nextPrevPrefixDateRange}&${nextPrevPrefixPageSize}&${nextPrevPrefixOrder}&sort=process.name&before=malware%20writer&before=4d7afd81-26ec-47c0-9741-ae16d331f73d`
-          )
-          .set('kbn-xsrf', 'xxx')
-          .expect(200);
-        let valid: boolean = true;
-        (body.alerts as AlertData[]).forEach((alert) => {
-          if (alert.process?.name > 'malware writer') {
-            valid = false;
-          }
-        });
-        expect(valid).to.eql(true);
-      });
-
-      it('alerts api should return data using `before` on undefined primary sort values by custom sort parameter, descending', async () => {
-        const { body } = await supertest
-          .get(
-            `/api/endpoint/alerts?${nextPrevPrefixDateRange}&${nextPrevPrefixPageSize}&order=desc&sort=${NULLABLE_EVENT_FIELD}&before=&before=${nullableEventId}&empty_string_is_undefined=true`
-          )
-          .set('kbn-xsrf', 'xxx')
-          .expect(200);
-
-        let lastSeen: string | undefined = 'zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz';
-        let valid: boolean = true;
-
-        for (const alert of body.alerts) {
-          const entityId = alert.process?.parent?.entity_id;
-          if (entityId === undefined && alert.event.id > nullableEventId) {
-            valid = false;
-          }
-          if (entityId !== undefined && lastSeen !== undefined && entityId > lastSeen) {
-            valid = false;
-          } else {
-            lastSeen = entityId;
-          }
-        }
-
-        expect(valid).to.eql(true);
-      });
-
-      it('alerts api should return data using `before` on undefined primary sort values by custom sort parameter, ascending', async () => {
-        const { body } = await supertest
-          .get(
-            `/api/endpoint/alerts?${nextPrevPrefixDateRange}&page_size=25&order=asc&sort=${NULLABLE_EVENT_FIELD}&before=&before=${nullableEventId}&empty_string_is_undefined=true`
-          )
-          .set('kbn-xsrf', 'xxx')
-          .expect(200);
-
-        let lastSeen: string | undefined = '1';
```

Comment:
Which tests are flaky? I'm only seeing one failure which is consistent in this test and can be fixed by changing this from `= '1'` to `= undefined`

Reply:
The other tests I'm removing have uuids in the URL and I'm not seeing where we're pinning the seed for generator so those could probably fail too.

Reply:
https://github.com/elastic/kibana/pull/69597/files#diff-2524267a6a848650421c834b7b4ab0f8L84

Reply:
Ah I missed that, well if we want to fix and add back that's fine.